_docs_v7/Slope-Limiters-and-Shock-Resolution.md

```diff
@@ -0,0 +1,205 @@
+
+The inclusion of a slope limiter into a TVD scheme accomplishes this idea. 
+
+
+
+<!-- ??? do we need to include def of venkat function??? -->
+<!-- It would make more sense to include the limiters in the section above -->
+
+
+## Available Limiter Options
+
+The field `SLOPE_LIMITER_FLOW` in the `.cfg` file specifies which limiter to use. Note that this option is only used if `MUSCL_FLOW = YES` (which specifies to use a second-order method).
+The [Laminar Cylinder](https://su2code.github.io/tutorials/Laminar_Cylinder/) shows an example of this.
+The [Turbulent Flat Plate example](https://su2code.github.io/tutorials/Turbulent_Flat_Plate/) sets `SLOPE_LIMITER_TURB`, which is used for the turbulence equations, rather than for the flow equations.
+More possible applications of limiters are listed below.
+
+
+### Slope Limiter Fields
+
+| Configuration Field | Description | Notes |
+| --- | --- | --- |
+| `SLOPE_LIMITER_FLOW` | Flow equations | Need `MUSCL_FLOW = YES` |
+| `SLOPE_LIMITER_TURB` | Turbulence equations | Need `MUSCL_TURB = YES` |
+| `SLOPE_LIMITER_SPECIES` | Species evolution equations | Need `MUSCL_SPECIES = YES` |
+| `SLOPE_LIMITER_ADJFLOW` | Adjoint flow equations | Need `MUSCL_ADJFLOW = YES` |
+| `SLOPE_LIMITER_ADJTURB` | Adjoint turbulence equations | Need `MUSCL_ADJTURB = YES` |
+
+
+<!-- 
+CLimiterDetails.hpp:
+computeLimiters.hpp uses computeLimiters_impl.hpp and really just sets the limiter
+computeLimiters_impl.hpp passes computation of field values (cell averages), gradients, and  to the limiter
+also check out
+computeGradientsGreenGauss.hpp (and the least squares one)
+ -->
+
+The `SLOPE_LIMITER_` options above may each be changed to use different limiters, which are listed and explained below.
+
+**Note:** the Discontinuous-Galerkin methods (DG) / Higher-order methods (HOM) do not use limiters.
+
+
+
+### Available Limiters
+
+| Type | Description | Notes |
+| --- | --- | --- |
+| `NONE`                  | No limiter                                      |  |
+| `BARTH_JESPERSEN`       | Barth-Jespersen                                 |  |
+| `VENKATAKRISHNAN`       | Venkatakrishnan                                 |  |
+| `VENKATAKRISHNAN_WANG`  | Venkatakrishnan-Wang                            |  |
+| `SHARP_EDGES`           | Venkatakrishnan with sharp-edge modification    | This limiter should not be used for flow solvers |
+| `WALL_DISTANCE`         | Venkatakrishnan with wall distance modification | This limiter should not be used for flow solvers |
+| `VAN_ALBADA_EDGE`       | Van Albada (edge formulation)                   | This limiter has been observed to not output limiter values when called using VOLUME_OUTPUT |
+
+<!-- ??? Van Albada. It's possible that we actually have this backward and Van Albada is the only limiter that works for other solvers??? -->
+<!-- We're currently investigating if Barth-Jespersen is Venkat with K=0 -->
```

Comment:
Van Albada is only implemented for flow solvers.
Currently, it is. However, that is inconsistent with the literature http://tetra.mech.ubc.ca/ANSLab/publications/michalak2008.pdf